src-cli/witan/models/run_models.clj

```diff
@@ -167,14 +169,18 @@
    ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
    ;; Predicates
    :finish-looping?            {:var #'witan.models.dem.ccm.core.projection-loop/finished-looping?
-                                :params {:last-proj-year (:last-proj-year params)}
+                                :params {:last-proj-yr (:last-proj-yr params)}
                                 :pred? true}
    ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
    ;; Inputs
    :in-hist-popn                  {:var #'witan.models.run-models/resource-csv-loader-filtered
                                    :params {:gss-code gss-code
                                             :src (:historic-population inputs)
                                             :key :historic-population}}
+   :in-future-mort-trend          {:var #'witan.models.dem.ccm.models-utils/resource-csv-loader
+                                   :params {:gss-code gss-code
```

Comment:
Because I was passing it in before I realised this dataset wasn't split by GSS code, will remove now, good spot!